### PR TITLE
feat: configure decimal places in JsonInstanceWriter

### PR DESCRIPTION
**Pull request by kapil-agnihotri**
This configuration can be added in
`<export-config>` element of `.halex`
project file as
`<setting name="numOfDecimalPlaces">14</setting>`
ING-3332

### CODE REVIEW

File: io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceWriter.java [hunk 1]
```diff
@@ -71,6 +70,14 @@ public boolean isCancelable() {
 	@Override
 	protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 			throws IOProviderConfigurationException, IOException {
+		Integer numOfDecimalPlaces = getParameter("numOfDecimalPlaces").as(Integer.class);
```
Comment (stempler) [inline io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceWriter.java:73]: The new parameter should also be added to the provider's definition in the extension point to document it.

Comment (kapil-agnihotri) [reply io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceWriter.java:73]: Thank you Simon for the feedback. I have updated provider's definition.

File: io/plugins/eu.esdihumboldt.hale.io.json/plugin.xml [hunk 2]
```diff
@@ -30,6 +30,19 @@
          <contentType
                ref="eu.esdihumboldt.hale.io.json.gzip">
          </contentType>
+         <providerParameter
+               default="2"
```
Comment (stempler) [inline io/plugins/eu.esdihumboldt.hale.io.json/plugin.xml:34]: This looks strange - I don't think there is a `default` attribute on that element. Did you use the Eclipse editor to add this?

**Review by stempler — APPROVED** [review]

File: io/plugins/eu.esdihumboldt.hale.io.json/plugin.xml (hunk 2 above)
Comment (kapil-agnihotri) [reply io/plugins/eu.esdihumboldt.hale.io.json/plugin.xml]: True, I initially edited the xml  but then later modified it from the editor. Looks like this left the initial changes unaltered. Thank you Simon.

**Pull request merged**